lib/kvdb/kvdb_rparams.c

```diff
@@ -22,6 +22,7 @@
 #include <hse_util/storage.h>
 #include <hse_util/compiler.h>
 #include <hse_util/string.h>
+#include <hse_util/invariant.h>
```

Comment:
Can you just move the guts of invariant.h into assert.h so that we can have just one file?

Reply:
It is doable, but for another PR :). I will create a ticket in JIRA.

Reply:
NFSE-5082